src/test/java/org/utplsql/api/db/DynamicParameterListTest.java

```diff
@@ -0,0 +1,58 @@
+package org.utplsql.api.db;
+
+import oracle.jdbc.OracleConnection;
+import org.junit.jupiter.api.Test;
+
+import java.sql.CallableStatement;
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class DynamicParameterListTest {
+
+    @Test
+    void call_with_three_different_types() throws SQLException {
```

Comment:
You know if you which to create human readable name of a test you can use `@Display` annotation which will be displayed in the results instead of the method name

Reply:
I know, the upside of this approach is that I know immediately what the test does without first having to look for the `@Display` content.
It's a very good addition, though.